Formula/c/clutter.rb

```diff
@@ -25,7 +25,6 @@ class Clutter < Formula
 
   depends_on "gobject-introspection" => :build
   depends_on "pkg-config" => :build
-  depends_on "atk"
```

Comment:
For #154695 since `clutter` is disabled and we can't build bottles. Also, I don't think we can just switch to `at-spi2-core` as that is currently Linux-only until other PR is done. So, temporarily remove dependency and can add `at-spi2-core` afterward (should be able to drop in replacement after other PR).